### PR TITLE
Fix images rendered by application

### PR DESCRIPTION
**Pull request by aduth**
**Why**: Vite requires static assets to be imported. It works locally only because node_modules exists in the working directly. But this folder is not created in the built environments.

### CODE REVIEW

File: bin/generate-routes.js [hunk 1]
```diff
@@ -1,9 +1,8 @@
 import { mkdir, copyFile } from "fs/promises";
 import { join } from "path";
-import { ROUTES } from "../src/routes";
 
 Promise.all(
-  Object.keys(ROUTES).map(async (path) => {
+  ["/daily-auths-report/"].map(async (path) => {
```
Comment (aduth) [inline bin/generate-routes.js:5]: Not super happy about this, but sometimes ESBuild tries (and fails) to parse the SVG, and in other cases Node doesn't know how to handle the static asset imports.

```
> node -r esbuild-register bin/generate-routes.js

identity-reporting/node_modules/identity-style-guide/dist/assets/img/login-gov-logo.svg:1
<?xml version="1.0" encoding="UTF-8"?>
^

SyntaxError: Unexpected token '<'
```

```
> node -r esbuild-register bin/generate-routes.js

identity-reporting/node_modules/esbuild-register/dist/node.js:2699
        throw error;
        ^

Error: Cannot find module '../node_modules/identity-style-guide/dist/assets/img/login-gov-logo.svg?url'
```

There's probably a way to fix this.

File: src/header.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        </button>
+      </div>
+      <nav className="usa-nav">
+        <div className="usa-nav__inner">
+          <button className="usa-nav__close" type="button">
+            <img src={closeURL} alt="Close" />
+          </button>
+          <ul className="usa-nav__primary usa-accordion">
+            <li className="usa-nav__primary-item">
+              <Link href="/" className={path === "/" ? "usa-current" : undefined}>
```
Comment (aduth) [inline src/header.tsx:31]: Gets the current path into the header by rendering it _within_ the route, not as an ancestor of it.

File: src/page.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  path: string;
+  children?: ComponentChildren;
+  title: string;
+}
+
+function Page({ path, children, title }: PageProps): VNode {
+  return (
+    <>
+      <Header path={path} />
+      <h1>{title}</h1>
```
Comment (aduth) [inline src/page.tsx:14]: This addresses the comment here: https://github.com/18F/identity-reporting/pull/23#discussion_r710091303

Even better than rendering it per-route, imo.

File: src/header.tsx (hunk 2 above)
Comment (aduth) [reply src/header.tsx:31]: Hm, this works locally, but not in Federalist.

**Review by zachmargolis — APPROVED** [review]
LGTM

File: src/daily-auths-report.tsx [hunk 4]
```diff
@@ -249,7 +249,7 @@ function DailyAuthsReport(): VNode {
 
   return (
     <div ref={ref}>
-      <div className="usa-accordion usa-accordion--bordered margin-bottom-2">
+      <div className="usa-accordion usa-accordion--bordered margin-top-2 margin-bottom-2">
```
Comment (zachmargolis) [inline src/daily-auths-report.tsx:252]: I snuck in the `margin-top-2` while I was here

File: src/page.tsx [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  path: string;
+  children?: ComponentChildren;
+  title: string;
+}
+
+function Page({ path, children, title }: PageProps): VNode {
+  return (
+    <>
+      <Header path={path} />
+      <div className="grid-container">
```
Comment (zachmargolis) [inline src/page.tsx:14]: I *think* we want the grid container to not include the header? so I stuck it here

**Pull request merged**